e2e/testenv/infra/go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/containerd/log v0.1.0 // indirect
 	github.com/docker/cli v23.0.5+incompatible // indirect
 	github.com/docker/distribution v2.8.2+incompatible // indirect
-	github.com/docker/docker v23.0.13+incompatible // indirect
+	github.com/docker/docker v23.0.14+incompatible // indirect
```

Comment:
Why not `v23.0.15`, which is listed as fixing the vulnerability?
Same remark elsewhere.

Reply:
I find this confusing, and it is unclear to me from the list of commits involved in that version how it might address the CVE listed on ArtifactHub. But the more up-to-date we stay, the better...